packages/components/src/InputNumber/index.ts

```diff
@@ -1 +0,0 @@
-export { InputNumber, InputNumberRef } from "./InputNumber";
```

Comment:
This file becomes `index.tsx` and is more sophisticated to include the version shim.